addon/templates/components/nypr-brand-header.hbs

```diff
@@ -0,0 +1,22 @@
+{{#side-menu side="right" width="200px" class="nypr-brand-header-menu" as |burger|}}
+  {{#side-menu-toggle class="nypr-brand-header-button nypr-brand-header-close"}}
+    <span>NYPR Network</span>
+    {{nypr-svg className="icon nypr-brand-header-close-icon" icon="close"}}
+  {{/side-menu-toggle}}
+
+  <ul class="nypr-brand-header-menu-items">
+    <li><a href="https://wnyc.org">WNYC</a></li>
+    <li><a href="https://gothamist.com">Gothamist</a></li>
+    <li><a href="https://wnycstudios.org">WNYC Studios</a></li>
+    <li><a href="https://wqxr.org">WQXR</a></li>
+    <li><a href="https://newsounds.org">New Sounds</a></li>
+  </ul>
```

Comment:
I know you're not going to like this, but I'd prefer it if these were actually yielded as contextual components that the user specifies. What if WQXR wants their link on top?

Reply:
Aw man, c'mon, you trying to get my goat right now? Isn't this how you get into an options mess, increasing complexity for the illusion of flexibility? What if they want a different background color? What if someday somebody wants the menu on the left? I guess the question to be answered on the ordering would be: *do they* want their site name on top? And couldn't we find that out now? 

I hope they don't, because from a UX standpoint I think the header is acting like a common navigation bar that joins all the NYPR properties together, and I'd expect it to behave like a navigation bar would on a single domain site—with the ordering staying the same between sections.

And in the future if/when NYPR gets another property, seems like we would want to add one line to this file and have all the clients take the update rather than add a new link to five different clients, wouldn't we?

Reply:
I was really hoping for your goat on this one, but I guess it’ll have to wait for another day. Leave it as it is.